R/dtc_utils.R

```diff
@@ -0,0 +1,163 @@
+#' Is it one of the supported formats?
+#'
+#' [is_dtc_fmt()] takes a character vector of date/time formats and returns
+#' a logical indicating which are supported.
+#'
+#' @details
+#' This function works by checking if the values of `x` are one of the formats
+#' listed in column `fmt` of [dtc_formats].
+#'
+#' @param fmt A character vector.
+#'
+#' @returns A [logical][base::logical] vector of the same size of `fmt`.
+#'
+#' @examples
+#' sdtm.oak:::is_dtc_fmt(c("ymd", "y m d", "dmy", "HM", "H:M:S", "y-m-d H:M:S"))
+#'
+#' sdtm.oak:::is_dtc_fmt("y years m months d days")
+#'
+#' @keywords internal
+is_dtc_fmt <- function(fmt) {
```

Comment:
Do we use `is_dtc_fmt` anywhere?

Reply:
Not at the moment, but I should. I will use it such that it won't be an orphan function.

Reply:
Ended up removing `is_dtc_fmt()`, see commit message for explanation: https://github.com/pharmaverse/sdtm.oak/commit/8cc595a81bb9c14625641bd54e769eccdcffbbcb.